extensions/stackdriver/metric/registry.cc

```diff
@@ -80,16 +81,19 @@ getStackdriverOptions(const Wasm::Common::FlatNode& local_node_info,
     }
   }
 
-  auto ssl_creds_options = grpc::SslCredentialsOptions();
+  grpc::experimental::TlsChannelCredentialsOptions tls_options;
+  tls_options.set_max_tls_version(grpc_tls_version::TLS1_2);
```

Comment:
Is the intension to 1) only use TLS 1.2 or 2) avoid using TLS 1.3?
From the following code in the CL, the intension seems to be 2) but the title of the CL seems to indicate 1).

 tls_options.set_max_tls_version(grpc_tls_version::TLS1_2);

Reply:
It's the same. gRPC sets TLS 1.2 as a minimum version.